src/midi.cpp

```diff
@@ -86,12 +88,17 @@ static void HandleNoteOff(byte channel, byte note) { HandleNoteOn(channel, note,
 
 static void HandleControlChange(byte channel, byte data1, byte data2) {
 	leftDot();
-	if (channel == masterChannel) {
+
+	if((channel ==16)&&(data1==85)){EEPROM.update(3994,data2);// settings (set by online webMIDI tool)
+	settings=data2;
+	}
+
+	else if (channel == masterChannel) {
 		if (data1 == 59)
 			data1 = 32;
 
 		if (1 <= data1 && data1 <= 36) {
```

Comment:
```suggestion
                if (data1 == 1) { // modwheel
                    preset_data.lfo[0].depth = value << 123; // put something appropriate here
                    // or, alternatively:
                    modwheelToLfo = value; // I'd prefer this. Create a variable similar to velocityToLfo.
                }
		else if (2 <= data1 && data1 <= 36) {
```

Reply:
sorry I made the changes before I scrolled to your suggestions - will read all the review in future!